service.subtitles.subtitool/changelog.txt

```diff
@@ -8,3 +8,5 @@
 1.1.2 - by KianMax 26/03/2016
 - Manual Search Added
 
```

Comment:
I know this is nowhere noted, but I would like to suggest doing the changelog with the newest always on top and getting older the more you get down.